CMakePresets.json

```diff
@@ -283,6 +283,20 @@
         "BUILD_PERFORMANCE_TESTS": true,
         "BUILD_SAMPLES": true
       }
-    }
+    },
+    {
+      "name": "linux-default",
```

Comment:
Thank you so much for your contribution, I had been contemplating adding this functionality for a couple of weeks now and it makes me happy to see someone moving forward with this.

To be consistent with the patterns that were initially established for Windows, I think that the CMAKE_BUILD_TYPE and VCPKG_TARGET_TRIPLET should be removed from linux-default and split into two separate "linux-debug" and "linux-release" presets which derive from "linux-default". Then, from those two bases, we can then add the other variants that we use with different VCPKG_TARGET_TRIPLET values. 

Not all of these need to be populated immediately, but it lays a foundation for building out the matrix of available options.

Reply:
Sounds right, will add these changes.